packages/playwright-test/src/ci.ts

```diff
@@ -0,0 +1,91 @@
+/**
+ * Copyright (c) Microsoft Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { spawnAsync } from 'playwright-core/lib/utils/utils';
+
+const GIT_OPERATIONS_TIMEOUT_MS = 1_500;
```

Comment:
Historic context: last time we had issues with number separators since our transpiler didn't touch them, and they made code crash on Node 12 I think. I ended up reverting my usages of them.

Maybe something has changed in our pipelines so it works now?